moler/config/connections.py

```diff
@@ -50,7 +50,7 @@ def set_defaults():
 
 def register_builtin_connections(connection_factory, moler_conn_class):
     _register_builtin_connections(connection_factory, moler_conn_class)
-    if platform.system() == 'Linux':
+    if platform.system() in ['Linux', "FreeBSD", "Darwin", "SunOS"]:
         _register_builtin_unix_connections(connection_factory, moler_conn_class)
```

Comment:
else and warn (Ususpported system {}".format(platform.system()) ?

Reply:
done raise MolerException on unsupported systems